src/wallet/rpcwallet.cpp

```diff
@@ -2280,6 +2280,8 @@ UniValue getwalletinfo(const UniValue& params, bool fHelp)
             "  \"keypoolsize\": xxxx,        (numeric) how many new keys are pre-generated\n"
             "  \"unlocked_until\": ttt,      (numeric) the timestamp in seconds since epoch (midnight Jan 1 1970 GMT) that the wallet is unlocked for transfers, or 0 if the wallet is locked\n"
             "  \"paytxfee\": x.xxxx,         (numeric) the transaction fee configuration, set in " + CURRENCY_UNIT + "/kB\n"
+            "  \"seedfp\": \"uint256\",        (string) the BLAKE2b-256 hash of the HD seed\n"
+            "  \"masterkeyid\": \"<hash160>\", (string) the Hash160 of the hd master pubkey\n"
```

Comment:
`masterkeyid` should not be here and is removed in a later commit

Reply:
If you end up rebasing the PR, please squash the later commit into this one.